tests/test_extension_amber.py

```diff
@@ -169,3 +169,15 @@ def test_nested_class_method(self, snapshot, actual):
 
 class TestSubClass(TestClass):
     pass
+
+
+@pytest.mark.parametrize("parameter_with_dot", ("value.with.dot",))
+def test_parameter_with_dot(parameter_with_dot, snapshot):
+    assert parameter_with_dot == snapshot
+
+
+@pytest.mark.parametrize("parameter_1", ("foo",))
+@pytest.mark.parametrize("parameter_2", ("bar",))
+def test_doubly_parametrized(parameter_1, parameter_2, snapshot):
```

Comment:
I was about to implement something that just split on `[`, on closer inspection, it seemed better to traverse the node's parents.